releases/v20.1.0.md

```diff
@@ -0,0 +1,129 @@
+    </script>
+</div>
+
+## Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.0.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.0.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.1.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.0.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+## Docker image
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ docker pull cockroachdb/cockroach:v20.1.0
+~~~
+
+## Summary
+
+This section summarizes the most significant, user-facing changes in v20.1.0. For a complete list of features and changes, including bug fixes and performance improvements, see the [release notes](index.html#testing-releases) for previous testing releases.
+
+- [CockroachCloud](#cockroachcloud)
+- [Core features](#core-features)
+- [Enterprise features](#enterprise-features)
+- [Backward-incompatible changes](#backward-incompatible-changes)
+- [Known limitations](#known-limitations)
+- [Documentation](#documentation)
+
+<style>
+    table td:first-child {
+        min-width: 225px;
+    }
+</style>
+
+### CockroachCloud
+
+
+### Core features
+
+These features are freely available in the core version and do not require an enterprise license.
+
+Feature | Description
+--------|------------
+**Online Primary Key Changes** | The new [`ALTER TABLE ... ALTER PRIMARY KEY`](../v20.1/alter-primary-key.html) statement lets you change a table’s primary key with no interruption to data access. The old primary key is converted to a `UNIQUE` secondary index to help optimize the performance of queries that still filter on the old key. However, if this conversion is not desired, you can [drop and add a primary key constraint](../v20.1/add-constraint.html#drop-and-add-a-primary-key-constraint) instead.
+**Bulk Import Improvements** | The [`IMPORT`](../v20.1/import.html) and [`IMPORT INTO`](../v20.1/import-into.html) statements now support bulk importing [from Avro files](../v20.1/migrate-from-avro.html) and [from local storage](../v20.1/import.html#import-a-table-from-a-local-file). The former makes it easier to migrate from systems like Spanner that export data in the Avro format, and the latter makes it possible to import without cloud storage. The new [`cockroach nodelocal upload`](../v20.1/cockroach-nodelocal-upload.html) command can be used to upload a file to a node's external IO directory for local import.<br><br>Also, [paused](../v20.1/pause-job.html) imports, when [resumed](../v20.1/resume-job.html), now continue from their internally recorded progress instead of starting over.
```

Comment:
nit. You could always import from nodelocal, it's just now easier to get those files to nodelocal from your "local" client (previously you had to SSH into the node and `scp` or something the file over)

What are your feelings on the usage of the word "local"? Local implies to me the machine you're typing on--if you SSH into a node I wouldn't personally consider that local. Do you have any thoughts on that @lnhsingh ? We seem to use local a bunch of places so I don't want to necessarily trigger a wholesale change across docs.

Reply:
Thanks, @mwang1026. I made some changes. PTAL. 

I agree that we overload the term "local". Initially, it was used to just contrast running cockroach on your computer vs on a server. We've probably outgrown that. We're likely to take a fresh look at our getting started docs very soon in any case.